pallav/fashionTech/pallavBanka/parkingAssignment/Models/Slot.java

```diff
@@ -0,0 +1,21 @@
+package fashionTech.pallavBanka.parkingAssignment;
+
+import java.util.LinkedList;
+
+public class Slot{
+    LinkedList<String> occupied=new LinkedList();
+    boolean occupySlot(Integer i) {
+        if (occupied.get(i)==null) {
```

Comment:
For this type of operation, Map OR Array performs better, as read and write time is O(1) where as in List it's O(n).